corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -277,8 +277,10 @@ function Form(json) {
     };
 
     self.afterRender = function() {
-        // Once form has finished rendering, render all popovers
-        $('.js-form-container [data-toggle="popover"]').popover();
+        $(".help-text-trigger").click(function(event) {
+            var container = $(event.target).closest(".caption");
+            container.find("#help-text-modal").modal('show');
```

Comment:
Picky, but you still have an id that appears multiple times on the page. The code works, but it still isn't semantic. I'd change this to `.help-text-modal` or just remove `help-text-modal` altogether and select `.modal` here, since there's only going to be one modal in the `.caption`.